src/main/java/org/gridsuite/notification/server/NotificationWebSocketHandler.java

```diff
@@ -95,12 +97,18 @@ public NotificationWebSocketHandler(ObjectMapper jacksonObjectMapper, @Value("${
             return res;
         }).map(m -> {
             try {
+                Map<String, Object> headers = new HashMap<>();
+                headers.put(HEADER_TIMESTAMP, m.getHeaders().get(HEADER_TIMESTAMP));
+                headers.put(HEADER_STUDY_NAME, m.getHeaders().get(HEADER_STUDY_NAME));
+                if (m.getHeaders().get(HEADER_UPDATE_TYPE) != null) {
+                    headers.put(HEADER_UPDATE_TYPE, m.getHeaders().get(HEADER_UPDATE_TYPE));
+                }
+                if (m.getHeaders().get(HEADER_ERROR) != null) {
+                    headers.put(HEADER_ERROR, m.getHeaders().get(HEADER_ERROR));
+                }
```

Comment:
Are these tests really necessary ?

Reply:
maybe not, but why putting a null value in the map ?

Reply:
I think is better for the client who will have to test only if the value of a header is null.
Otherwise it have to be test also if the header exists.

Reply:
Done